src/markdown-pages/build-apps/permission-manage-apps.mdx

```diff
@@ -13,41 +13,124 @@ tags:
 
-The most important permissions factor is [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#user-type). A basic user has several restrictions related to their inability to access Full Stack Observability, and a full user theoretically has full abilities.
+</Intro>
 
-Basic users can build and serve their own Nerdpacks locally. They can also use public Nerdpacks that meet **both** of the following criteria:
+## Capabilities at a glance
 
-* New Relic must have specifically allowed basic users to use the Nerdpack
-* Someone else on the basic user's account must have already subscribed to the Nerdpack
+Your ability to manage and use Nerdpacks depends on your user's:
 
-Nerdpacks that basic users can use are rare.
+- Model
+- Type
+- Role
 
-Basic users can't:
+Whether you're on the [original user model](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/) or the [New Relic One user model](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/), review the tables below for a summary of user Nerdpack capabilities.
 
-* Publish the Nerdpacks they build
-* Tag their Nerdpacks
-* Subscribe an account to a Nerdpack
-* Use private Nerdpacks (including their own)
-* Use public Nerdpacks that New Relic hasn't specifically allowed them to use
+### Original user model
 
-Full users can use any Nerdpacks that the account they're in has been subscribed to, whether built by New Relic or others. Full users theoretically have Nerdpack management permissions, but there may be restrictions related to custom role assignments (see below).
+If your organization was created before July 30, 2020 and you haven't transitioned to our New Relic One pricing model, you're on the [original pricing plan](https://docs.newrelic.com/docs/accounts/original-accounts-billing/product-pricing/product-based-pricing/). Capabilities for our [original user model](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/) differ from those of our [new user model](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/):
 
-## Role-related restrictions
+| | Basic user | Full user without the Nerdpack Manager add-on role | Full user with the Nerdpack Manager add-on role | Owner/Admin user |
+|:--|:-:|:-:|:-:|:-:|
+| Serve Nerdpacks | yes | yes | yes | yes |
+| Publish Nerdpacks | no | no | yes | yes |
+| Subscribe to Nerdpacks | no | no | yes | yes |
+| Tag Nerdpacks | no | no | yes | yes |
+| Use Nerdlets or visualizations created by the user's accounts | no | yes | yes | yes |
+| Use Nerdlets created by New Relic | no* | yes | yes | yes |
+| Use visualizations created by New Relic | no | yes | yes | yes |
```

Comment:
Since we capitalize `Nerdlets` should we also capitalize `Visualizations`?

Reply:
Unfortunately not, at least not according to the naming guides I've been given. "Visualization" is a common noun, whereas "Nerdlets" is proper because it's a branded name.